lib/queries.js

```diff
@@ -283,7 +291,7 @@ export async function getAllAbbKboOrganizations() {
           FILTER (?ovoNotation IN ("OVO-nummer"@nl, "OVO-nummer"))
         }
        }
-     }
+     } GROUP BY ?abbOrg
```

Comment:
We only need one ABB organization per KBO number. 
Utilizing GROUP BY helps in condensing the result length from 17,084 to 1,682.